components/Header/styles.js

```diff
@@ -1,4 +1,5 @@
 import styled from 'styled-components';
```

Comment:
Also I have refactored your code a bit to make it I'd say better (and easier to work later).

```js
import styled from 'styled-components';
import { tablet } from '../../styles/breakpoints';

export const StyledHeader = styled('header')`
  width: 100%;
  position: relative;
  height: 26rem;

  @media (${tablet}) {
    height: 36rem;
  }
`;

export const LogoContainer = styled('div')`
  position: absolute;
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;
  padding: 2rem 3rem;
  height: 9rem;

  @media (${tablet}) {
    padding: 3rem;
    height: 12rem;
  }
`;

export const StyledLogo = styled('img')`
  max-width: 100%;
  max-height: 100%;
`;

export const StyledBanner = styled('img')`
  height: 50rem;
  position: absolute;
  left: 50%;
  transform: translateX(-50%);
  image-rendering: pixelated;

  @media (${tablet}) {
    height: 70rem;
  }
`;

```

Also I have added a main element around the content that is just that but it makes the content be above the header itself

```js
import styled from 'styled-components';

const Main = styled('main')`
  position: relative;
`;

export default Main;
```